xmodule/modulestore/mixed.py

```diff
@@ -295,6 +295,11 @@ def get_course_summaries(self, **kwargs):
                     log.warning(
                         "Modulestore %s have duplicate courses %s; skipping from result.", store, course_id
                     )
+                # Skip course if it's from old mongo modulestore.
+                elif course_id.deprecated:
```

Comment:
Let's continue to return the course summary for now. We should be able to grab that information from the root CourseBlock anyhow.